tests/Dfe.PlanTech.Web.E2ETests/cypress/e2e/dynamic-page-validator/dpv-validators/completion-tags-validator.js

```diff
@@ -0,0 +1,49 @@
+import { CleanText } from "../../../helpers/text-helpers";
+import { getSubmissionTimeText } from "../dpv-helpers/time-helpers";
+
+
+export const selfAssessmentPageNewRecommendation = (section, maturity) => {
+    const time = new Date();
+    const timePlusOneMinute = new Date(time).setMinutes(time.getMinutes() + 1);
+    const lateTime = new Date(timePlusOneMinute);
+
+    const matchingRecommendation = section.recommendation.intros.find(
```

Comment:
`validateRecommendationForMaturity` in `recommendations-validator.js` already performs this exact same function on line 14. Could we pass in recommendation that has already been found to this function instead?

Reply:
Good shout, thanks. I've moved the error handling up to the parent as well.